pages/common/samtools.md

```diff
@@ -0,0 +1,28 @@
+# samtools
+
+> Tools for handling high-throughput sequencing (genomics) data.
+> Reading/writing/editing/indexing/viewing of data in SAM/BAM/CRAM format.
+
+- Convert a SAM file to BAM (use - for stdin):
+
+`samtools view -bS {{input.sam}} > {{output.bam}}`
```

Comment:
`-S` is a separate option. It specifies `SAM` input format. In the latest versions of samtools this flag is actually ignored, the format is autodetected from the input filename extension. However, a range of samtools versions are still common so I typically include it in scripts for backwards compatibility.

`-b` specifies that the output should be in `BAM` format instead of the default `SAM` for `stdout`. This isn't needed if using `-o {{output.bam}}` (autodetected by file extension again).